recipe/meta.yaml

```diff
@@ -15,6 +15,16 @@ build:
     - COPY "%RECIPE_DIR%\\upload_or_check_non_existence.py" "%SCRIPTS%\\upload_or_check_non_existence.py"                # [win]
     - cp "${RECIPE_DIR}/upload_or_check_non_existence.py" "${PREFIX}/bin/upload_or_check_non_existence"                  # [unix]
 
+requirements:
+  run:
+    - conda
+    - conda-env
+    - conda-build
+    - jinja2
+    - anaconda-client
+    - conda-build ==2
+    - vs2008_express_vc_python_patch  # [win]
```

Comment:
`and py27`?

Reply:
We don't know which python version is going to be used in the recipe, so this is installed everywhere for windows as is the case in master

Reply:
Yeah, this is actually no different than what we are already doing here. Though agree it would be nice if we didn't always have to do it. Think it has been easier for everyone to just leave it as is.